hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -2957,6 +2958,17 @@ public String inlineLiteral(String literal) {
 		return String.format( "\'%s\'", escapeLiteral( literal ) );
 	}
 
+	/**
+	 * Check whether the JDBC {@link java.sql.Connection} supports creating LOBs via {@link Connection#createBlob()} or {@link Connection#createBlob()}.
```

Comment:
`{@link Connection#createBlob()} or {@link Connection#createBlob()}`

both are the same :). I think you mean Clob for one of them?

Reply:
Good catch.